webhooks/webhooks.helper.ts

```diff
@@ -0,0 +1,300 @@
+            recordName);
+    }
+
+    static async clickRecordEditIcon(recordName: string) {
+        const editIcon = WebhooksPage.getRecordEditIcon(recordName);
+        StepLogger.subVerification(`Click Edit for ${recordName}`);
+        // normal click does not work for this element, so we use clickUsingJS
+        await ElementHelper.clickUsingJs(editIcon);
+    }
+
+    static async clickRecordDeleteIcon(recordName: string) {
+        const deleteIcon = WebhooksPage.getRecordDeleteIcon(recordName);
+        StepLogger.subVerification(`Click Delete for ${recordName}`);
+        // normal click does not work for this element, so we use clickUsingJS
+        await ElementHelper.clickUsingJs(deleteIcon);
+    }
+
+    static async deleteRecordAndOK(recordName: string) {
+        await this.clickRecordDeleteIcon(recordName);
+        await this.clickOK();
+    }
+
+    static async loginAndCreateWebhookRecord(url: string, filter: string, name: string) {
+        await ManuscriptLoginPageHelper.loginToManuscript();
+        await ManuscriptHomePageHelper.verifyPageLoaded();
+        await ManuscriptHomePageHelper.mouseHoverAvatarAndClickWebhooks();
+        await this.verifyPage();
+        await this.clickAddNewHook();
+        await this.enterRequiredFieldsAndOK(url, filter, name);
+        await this.verifyWebhookRecord(name);
+    }
+
+    static async verifyEditHookDialog() {
+        StepLogger.subVerification('Verify Edit Hook Dialog is displayed');
+        await ExpectationHelper.verifyDisplayedStatus(labelsPO.editHook, textConst.editHook);
+    }
+
+    static async verifyDeleteHookDialog() {
+        StepLogger.subVerification('Verify Delete Hook Dialog is displayed');
+        await ExpectationHelper.verifyDisplayedStatus(labelsPO.deleteHook, textConst.deleteHook);
+    }
+
+    static async clickViewLog() {
+        StepLogger.subStep('Click on View Log Link');
+        await PageHelper.click(confLinksPO.viewLog);
+        await CommonPageHelper.switchToCreateFrame();
+    }
+
+    static async clickImport() {
+        StepLogger.subStep('Click on Import Link');
+        await PageHelper.click(confLinksPO.import);
+        await CommonPageHelper.switchToCreateFrame();
+    }
+
+    static async clickExport() {
+        StepLogger.subStep('Click on Export Link');
+        await ElementHelper.clickUsingJs(confLinksPO.export);
+    }
+
+    static async verifyXMLImportPage() {
+        StepLogger.subVerification('Verify XML Import Page is displayed');
+        await ExpectationHelper.verifyDisplayedStatus(labelsPO.xmlImportHeader, textConst.xmlImport);
+    }
+
+    static async uploadFile(filePath: string) {
+        StepLogger.subStep(`Upload file at path: ${filePath}`);
+        const chooseFileButton = WebhooksPage.chooseFile;
+        await UploadHelper.uploadFile(chooseFileButton, filePath);
+    }
+
+    static getUploadFilePath(fileName: string) {
+        const uploadFilePath = __dirname + '/' + fileName;
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/davidgunadi/hello-world/pullRequest?prid=8400130)
```suggestion
        const uploadFilePath = __dirname + "/" + fileName;
```